Library/Formula/png2ico.rb

```diff
@@ -1,21 +1,25 @@
-require 'formula'
-
 class Png2ico < Formula
-  homepage 'http://www.winterdrache.de/freeware/png2ico/'
-  url 'http://www.winterdrache.de/freeware/png2ico/data/png2ico-src-2002-12-08.tar.gz'
-  sha1 '955004bee9a20f12b225aa01895762cbbafaeb28'
+  homepage "http://www.winterdrache.de/freeware/png2ico/"
+  url "http://www.winterdrache.de/freeware/png2ico/data/png2ico-src-2002-12-08.tar.gz"
+  sha1 "955004bee9a20f12b225aa01895762cbbafaeb28"
   revision 1
 
-  depends_on 'libpng'
+  depends_on "libpng"
 
   # Fix build with recent clang
   patch :DATA
 
   def install
-    inreplace 'Makefile', 'g++', '$(CXX)'
+    inreplace "Makefile", "g++", "$(CXX)"
     system "make", "CPPFLAGS=#{ENV.cxxflags} #{ENV.cppflags} #{ENV.ldflags}"
-    bin.install 'png2ico'
-    man1.install 'doc/png2ico.1'
+    bin.install "png2ico"
+    man1.install "doc/png2ico.1"
+  end
+
+  test do
+    curl "-O", "http://www.google.com/images/icons/product/googlemail-32.png"
+    system "#{bin}/png2ico", "out.ico", "googlemail-32.png"
```

Comment:
Then I need to replace current fixture png to a 8*8 size png file. So how do you guys generate a png file to avoid copyright issue?